plugin/public/reearth.yml

```diff
@@ -25,4 +25,12 @@ extensions:
       defaultLocation:
         zone: outer
         section: left
-        area: bottom
\ No newline at end of file
+        area: bottom
+  - id: geolocation
+    name: My Location
+    type: widget
+    widgetLayout:
+      defaultLocation:
+        zone: outer
+        section: right
+        area: middle
```

Comment:
Should be inner right middle. If it (and the navigator, etc) are in the outer zone, the timeline at the bottom of the screen can't stretch all the way to the right.

Reply:
ok ,I get that , done

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6